school_management/tests/test_school_student.py

```diff
@@ -0,0 +1,90 @@
+from datetime import date
+
+from dateutil.relativedelta import relativedelta
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestSchoolStudent(TransactionCase):
+    def setUp(self):
+        """Created a student,class,payments records and two subject records #T00475"""
+        self.subject_1 = self.env["school.student.subjects"].create(
+            {"subjects_name": "Maths"}
+        )
+        self.subject_2 = self.env["school.student.subjects"].create(
+            {"subjects_name": "Science"}
+        )
+        self.payment_1 = self.env["school.student.fees"].create({"paid_fees": 200})
+        self.class_1 = self.env["school.student.classes"].create(
+            {"division": "divisionA"}
+        )
+        self.student_1 = self.env["school.student"].create(
+            {
+                "student_first_name": "Test1",
+                "student_last_name": "Test1",
+                "rollno": 1,
+                "date_of_birth": date.today() - relativedelta(years=15),
+                "address_student": "TEST",
+                "standard": "standard1",
+                "subject_list_ids": [
+                    (6, 0, [self.subject_2.id, self.subject_1.id]),
+                ],
+                "fees_students_ids": [
+                    (6, 0, [self.payment_1.id]),
+                ],
+                "division_of_students_id": self.class_1.id,
+            }
+        )
+        super(TestSchoolStudent, self).setUp()
+
+    def test_compute_age(self):
+        """tests the compute age method #T00475"""
+        self.assertEqual(self.student_1.age_of_student, 15, "Fail")
```

Comment:
In the test message mention why the test case failed not just "Fail". Do the same for all the test methods.
A message should be like "Student age did not match!"